elasticsearch/init.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash -e
+#
+# Copyright 2018 Red Hat, Inc. and/or its affiliates
+# and other contributors as indicated by the @author tags.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source "logging"
+
+LOG_FILE=${LOG_FILE:-elasticsearch_connect_log.txt}
+
+wait_for_port_open() {
+    rm -f $LOG_FILE
+    # test for ES to be up first and that our SG index has been created
+    info "Checking if Elasticsearch is ready on $ES_REST_BASEURL"
+    while ! response_code=$(es_util --query=/ \
```

Comment:
`es_util` hits `$ES_BASE`, not `$ES_REST_BASEURL`

Reply:
removed ref.  not significant though maybe it is if the script is running in a container to access some random ES node instead of local?